app/controllers/home_controller.rb

```diff
@@ -1,5 +1,13 @@
 class HomeController < ApplicationController
   def index
     skip_policy_scope
+
+    if current_user.admin?
+  		redirect_to admin_outreach_events_path
+		elsif current_user.facilitator?
+  		redirect_to facilitator_outreach_events_path
+		elsif current_user.student?
+  		redirect_to outreach_events_path
+		end
   end
```

Comment:
Instead of redirecting from home#index, we can remove the `root "home#index"` from `routes.rb` and add:

```
root :to => 'admin/outreach_events#index', :constraints => lambda { |request| request.env['warden'].user && request.env['warden'].user.admin? }
  root :to => 'outreach_events#index', :constraints => lambda { |request| request.env['warden'].user && request.env['warden'].user.student?|| request.env['warden'].user.facilitator? }
```